src/psyclone/tests/domain/lfric/transformations/dynamo0p3_transformations_test.py

```diff
@@ -7441,5 +7457,4 @@ def fake_validate(_1, _2, options=None):
                 else:
                     trans.validate(loop)
             assert "validate test exception" in str(err.value), \
-                "{0}.validate() does not call LoopTrans.validate()".format(
-                    name)
+                f"{name}.validate() does not call LoopTrans.validate()"
```

Comment:
Thanks a lot for all the code cleanup (f-strings, imports not in toplevel etc)!!